### DISCUSSION TITLE
Problem with settings switching themselves

### DISCUSSION THREAD

**Discussion by Stgurgel** [SamDel/ChromeCast-Desktop-Audio-Streamer#124] [Q&A] [answered]
I followed someones advice to install a virtual sound out source (VB-audio virtual cable) to support my use case, and have CCDAS use that for recording. Works great. I have my sound out on computer set to the same and get silent computer  speakers. But when I switch my sound out on the computer back to speakers CCDAS only stays silent for a while before switching its own recording settings to current output, and I'm back to sound from two places and not in sync.
How do I stop CCDAS from trying to be smart and accepting the settings I make?
Regards/ Gurgel from sweden (Lenovo yoga AMD laptop)

**Comment by SamDel** [comment, marked as answer]
> How do I stop CCDAS from trying to be smart and accepting the settings I make?

You can't in the latest version. Maybe I'll add a solution for it later.

The code to automatically switch to the new default device is added in version 3.6. [Version 3.5](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/releases/tag/v3.5) should work for you. I'll come back to you later...

**Reply by Stgurgel** [reply]
Thank you for an excellent solution for my usecase
